extensions/usAnnotation/src/getCommandsModule.ts

```diff
@@ -0,0 +1,361 @@
+import { UltrasoundAnnotationTool, Enums as csToolsEnums } from '@cornerstonejs/tools';
+import { Types as OhifTypes } from '@ohif/core';
+import { setShowPercentage } from './getCustomizationModule';
+import { eventTarget, triggerEvent, utilities } from '@cornerstonejs/core';
+
+const { transformWorldToIndex } = utilities;
+
+/**
+ * Retrieves the DICOM instance associated with a specific imageId
+ * @param services - The OHIF services object
+ * @param imageId - The image ID to find the instance for
+ * @returns The DICOM instance object or undefined if not found
+ */
+function getInstanceByImageId(services: AppTypes.Services, imageId: string) {
+  const activeDisplaySets = services.displaySetService.getActiveDisplaySets();
+  const displaySet = activeDisplaySets.find(displaySet => displaySet?.imageIds?.includes(imageId));
+  return displaySet?.instance;
+}
+
+/**
+ * Creates and returns the commands module for ultrasound annotation
+ * @param params - Extension parameters including servicesManager and commandsManager
+ * @returns The commands module with actions and definitions
+ */
+function commandsModule({
+  servicesManager,
+  commandsManager,
+}: OhifTypes.Extensions.ExtensionParams): OhifTypes.Extensions.CommandsModule {
+  const { viewportGridService, toolGroupService, cornerstoneViewportService } =
+    servicesManager.services as AppTypes.Services;
+
+  const actions = {
+    /**
+     * Switches the active ultrasound annotation type
+     * @param options - Object containing the annotationType to switch to
+     */
+    switchUSAnnotation: ({ annotationType }) => {
```

Comment:
I think `switchUltrasoundBlinePluralAnnotationType` or something like it would work.